pkg/ddc/juicefs/transform_fuse.go

```diff
@@ -411,3 +422,126 @@ func (j *JuiceFSEngine) genFormatCmd(value *JuiceFS, config *[]string) {
 	cmd := append([]string{common.JuiceCliPath, "auth"}, args...)
 	value.Configs.FormatCmd = strings.Join(cmd, " ")
 }
+
+func (j *JuiceFSEngine) getQuota(v string) (int64, error) {
+	q, err := resource.ParseQuantity(v)
+	if err != nil {
+		return 0, fmt.Errorf("invalid quota %s: %v", v, err)
+	}
+	qs := q.Value() / 1024 / 1024 / 1024
+	if qs <= 0 {
```

Comment:
I think we also need to check if the value has overflowed int64 range:

```golang
	if qs > math.MaxInt64 {
		return 0, fmt.Errorf("quota %s is too large", v)
	}
```

Reply:
done

Reply:
q.Value() is int64, it may not be more than MaxInt64